File: WowAutoApp.Web.Client/src/app/core/enums/OwnerType.ts
export enum OwnerType {
    SelfOrSpouse = 0,
    Landlord  = 1,
    Relative = 2,
    Mortgage = 3,
    Others = 4
}

export namespace OwnerType {

    export function values() {
        return Object.keys(OwnerType).filter(
            (type) => isNaN(<any>type) && type !== 'values'
        );
    }
}   

File: Libraries/WowAutoApp.Core/Domain/Enums/OwnerType.cs
using System.ComponentModel;

namespace WowAutoApp.Core.Domain.Enums
{
    /// <summary>
    /// Credit application residence owner type
    /// </summary>
    public enum OwnerType
    {
        [Description("Self or Spouse")]
        SelfOrSpouse = 0,
        [Description("Landlord")]
        Landlord = 1,
        [Description("Relative")]
        Relative = 2,
        [Description("Mortgage")]
        Mortgage = 3,
        [Description("Others")]
        Others = 4
    }
}